src/main/scala/com/fulcrumgenomics/util/Sequences.scala

```diff
@@ -55,25 +58,73 @@ object Sequences {
     * @param s a sequence
     * @return a tuple of (0-based index, length) of the longest homopolymer in the string
     */
+  @deprecated("Use maxPolyX instead", since="0.7.0")
   def longestHomopolymer(s: String): (Int, Int) = {
-    val xs = s.toUpperCase
-    val len = xs.length
-    assert(len > 0, "Cannot compute longest homopolymer of a zero length string")
+    assert(s.length > 0, "Cannot compute longest homopolymer of a zero length string")
+    val result = maxPolyX(s)
+    (result.offset, result.length)
+  }
+
+
+  case class OffsetAndLength(offset: Int, length: Int)
+
+  /**
+    * Returns the offset (0-based) and length of the longest homopolymer in the sequence. In the case
+    * that there are multiple homopolymers of the same length, the earliest one is returned.
+    *
+    * @param s a DNA or RNA sequence
+    * @return the offset and length of the longest homopolymer
+    */
+  def maxPolyX(s: String) : OffsetAndLength = {
```

Comment:
Two thoughts on naming (apply to maxDinux as well):
1) I think longest is clearer than max.  Longest can only really be interpreted one way.  Max could be interpreted to mean "longest single", or "most abundant" such that in a sequence like `ACACGTGTGTACAC` you might say `AC` is the "max dinuc" because there is more of it.
2) Homopolymer is a well defined term whereas  `polyX` isn't really.

Reply:
Agreed, fixed.